### PR TITLE
AnkerMake profile updates

### PR DESCRIPTION
**Pull request by just-trey**
This update provides many more profile options and includes the new AnkerMake M5C printer. These profiles are based on some thoroughly tested and popular Prusa Slicer profiles I maintain for the community at https://github.com/Ankermgmt/prusaslicer-ankermake-ce-profiles. If you have any questions or see any improvements needed, please let me know.

### CODE REVIEW

File: resources/profiles/Anker/process/0.15mm Optimal @Anker.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "type": "process",
+    "setting_id": "GPGP_Anker_003",
```
Comment (XunZhangBambu) [inline resources/profiles/Anker/process/0.15mm Optimal @Anker.json:3]: setting_id is not valid

Comment (SaltWei) [reply resources/profiles/Anker/process/0.15mm Optimal @Anker.json:3]: @XunZhangBambu Please assign a right one

Comment (SaltWei) [reply resources/profiles/Anker/process/0.15mm Optimal @Anker.json:3]: @XunZhangBambu Please assign a right one

File: resources/profiles/Anker/machine/Anker M5 0.4 nozzle.json [hunk 2]
```diff
@@ -1,29 +1,28 @@
 {
     "type": "machine",
-    "setting_id": "GM_Anker_000",
+    "setting_id": "GM_Anker_001",
```
Comment (XunZhangBambu) [inline resources/profiles/Anker/machine/Anker M5 0.4 nozzle.json:3]: duplicate setting_id, GM_Anker_001 has been used by Anker M5C

File: resources/profiles/Anker/process/0.15mm Fast @Anker.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "enable_overhang_speed": "1",
+    "bridge_flow": "0.85",
+    "initial_layer_print_height": "0.15",
+    "layer_height": "0.15",
+    "raft_contact_distance": "0.15",
+    "top_shell_layers": "6",
+    "support_bottom_z_distance": "0.15",
+    "support_top_z_distance": "0.15",
+    "top_surface_pattern": "rectilinear",
+    "bottom_surface_pattern": "rectilinear",
```
Comment (XunZhangBambu) [inline resources/profiles/Anker/process/0.15mm Fast @Anker.json:18]: it should be zig-zag，Rectilinear  is a label ,the real value  is zig-zag

File: resources/profiles/Anker/machine/Anker M5 0.4 nozzle.json (hunk 2 above)
Comment (just-trey) [reply resources/profiles/Anker/machine/Anker M5 0.4 nozzle.json:3]: changed as requested

File: resources/profiles/Anker/process/0.15mm Fast @Anker.json (hunk 3 above)
Comment (just-trey) [reply resources/profiles/Anker/process/0.15mm Fast @Anker.json:18]: changed as requested

File: resources/profiles/Anker/process/0.15mm Optimal @Anker.json (hunk 1 above)
Comment (just-trey) [reply resources/profiles/Anker/process/0.15mm Optimal @Anker.json]: changed as requested

**Comment by just-trey** [comment]
@XunZhangBambu changes made as requested, plus removal of a formatted file you probably did not want. Can you please re-review?

**Comment by XunZhangBambu** [comment]
@just-trey  Thanks for your effort. I will review it later

File: resources/profiles/Anker/process/0.15mm Fast @Anker.json (hunk 3 above)
Comment (XunZhangBambu) [reply resources/profiles/Anker/process/0.15mm Fast @Anker.json]: There are still some existing rectilinear patterns, and these values will cause the preset to fail to load properly. Could you please make the necessary modifications ?

**Comment by XunZhangBambu** [comment]
@just-trey  Thanks for your PR! I have reviewed your PR again  and provided some modification suggestions . If you need help or have any questions, feel free to ask me.

File: resources/profiles/Anker/process/0.15mm Fast @Anker.json (hunk 3 above)
Comment (just-trey) [reply resources/profiles/Anker/process/0.15mm Fast @Anker.json]: @XunZhangBambu, there should be no more rectilinear values present now. One thing to note is there are A LOT of other profiles that use rectilinear.

**Comment by just-trey** [comment]
@XunZhangBambu  please look again and issuee should be resolved.

**Comment by XunZhangBambu** [comment]
@just-trey  I appreciate your efforts very much .I have already fix some  existing bugs in this profiles.Now the profile can run correctly in Bambu Studio. Would you prefer me to push the updated profile to your repository, or should I directly push it to our repository and @ you in the commit message?

**Comment by just-trey** [comment]
@XunZhangBambu I'm okay if you just @ me on the updates seems like that would be the fastest way.

**Comment by zzz-x** [comment]
@just-trey  Your modification can be seen in the latest version v1.7.4. Thanks very much for your efforts

**Comment by just-trey** [comment]
@zzz-x @XunZhangBambu you can close this pr. Thank you for including it. I do not think it is the profiles, as I them in a previous version before I opened the issue., but you may want to look at #2326 there is something weird going on.

**Comment by XunZhangBambu** [comment]
ok, i will check it later

**Pull request closed without merging**